Shopify/shopify-cli
# Bump theme-check version to v1.11.0

### WHY are these changes introduced?

`theme-check` requires a newer liquid version to support inline comments `{% # this is an inline comment %}`.

### WHAT is this pull request doing?

- Bumps the theme-check version requirement to v1.11.0
- Bumps the min ruby version to 2.6
- Removes the 2.6 ruby version from the github workflow files
- Removes the 2.6 references in the linux vagrant scripts

### How to test your changes?

Include an inline comment in a file in a theme and make sure theme-check doesn't report a SyntaxError.

### Post-release steps

Might require an announcement about dropped ruby 2.6 support.

### Update checklist
